get/application.go

```diff
@@ -75,7 +80,7 @@ func printApplication(apps []apps.Application, get *Cmd, out io.Writer, header b
 
 func printCredentials(creds []appCredentials, get *Cmd, out io.Writer) error {
 	if get.Output == yamlOut {
-		return format.PrettyPrintObject(creds, format.PrintOpts{Out: out})
+		return format.PrettyPrintObjects(creds, format.PrintOpts{Out: out})
```

Comment:
Unrelated to this change but just got me thinking that we should probably detect if we're in a terminal within `PrettyPrintObjects` and then output the yaml object without any color codes so it's actually usable for scripting.

Reply:
You are actually right. This would make sense.

just FYI: This change was only done as I preferred the output in multiple yaml documents instead of showing a sequence directly. Not sure if this actually is the right way (or whats better for scripting). In the end I just want them to be consistent between showing basic auth credentials and the dns details.

Reply:
I add the terminal handling part. All writes to a non-terminal are now made without colour.